### ISSUE TITLE
Shift+Tab to unindent code triggers Steam Overlay instead

### ISSUE THREAD

**Issue by mbowersjr** [big-ab-games/robo-instructus#244]
Pressing <kbd>Tab</kbd>to indent a selected block of code or the current line of code works well, just as it would in an IDE such as Visual Studio. However, I'm constantly pressing <kbd>Shift</kbd> + <kbd>Tab</kbd>, expecting it to unindent selected code or the current line, similar to the IDE.

In this case though, <kbd>Shift</kbd> + <kbd>Tab</kbd> is the default key combination to open the Steam Overlay. I realize this isn't any fault of the game, but would there be any convenient solution to this inconvenience?

**Comment by alexheretic** [comment]
Yep this isn't ideal, but difficult to fix with the steam hotkeys as they are. 

A workaround that's in place `ctrl shift tab` which is documented in the game `README.txt` file.
